5.6.1-alpine/run.sh

```diff
@@ -7,6 +7,7 @@ if [ "${1:0:1}" != '-' ]; then
 fi
 
 exec java -jar lib/sonar-application-$SONAR_VERSION.jar \
+  ${SONARQUBE_JAVA_ARGS} \
```

Comment:
This seems completely useless, because arguments specified for `docker run` as
```
docker run -p 9000:9000 -it sonarqube:5.6.1 argument
```
already will appended - see https://github.com/SonarSource/docker-sonarqube/blob/73534fd899d29eb1373ea073edbfe7a4074ede6e/5.6.1/run.sh#L15